run_tests.py

```diff
@@ -186,6 +194,39 @@ def prompt_user_to_run_tests(test_names):
             return
 
 
+def report_test_coverage_for_devices(tested_directories):
+    """
+    Report the ioc directories not tested
+
+    Args:
+        tested_directories (list): List of IOC boot directories generated by make_device_launchers_from_module
+
+    Returns:
+        None
+    """
+    # get names of iocs from ioc folder
+    iocs_dir = os.path.join(EPICS_TOP, "ioc", "master")
+    iocs = []
+    for dir in os.listdir(iocs_dir):
+        if os.path.isdir(os.path.join(iocs_dir, dir)):
+            iocs.append(dir)
+    iocs = [ioc.lower() for ioc in iocs]
```

Comment:
Could just do this here so you don't need to convert to a set later
```suggestion
    iocs = set(ioc.lower() for ioc in iocs)
```